### PR TITLE
seprate css file 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>12 o'clocks</title>
+    <link rel="shortcut icon" href="http://tdctokyo.org/eng/wp/wp-content/uploads/2010/08/1acc8a9dfa88b5ab93da8d26fd93a95c.jpg"/>
```
Comment (meiamsome) [inline index.html:6]: I'm not sure about this icon image - there are usually fairly strict requirements on favicons (Particularly that they are quite low resolution. Also, it'd be better if it was stored locally on the github.

File: index.html [hunk 2]
```diff
@@ -25,128 +26,6 @@
...
-        grid-row-end: 3;
-      }
-      #container .contributors {
-        grid-row-start: 3;
-        grid-column-start: 1;
-        grid-column-end: 3;
-      }
-    }
-
-    </style>
```
Comment (meiamsome) [inline index.html:149]: We need to link to the styles externally with a link element fro this to work

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:6]: I agree, I mainly just did this as a example

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html:149]: forgot to do that 😆

**Comment by meiamsome** [comment]
@rccji Perhaps reset up to cf02d83 & add the link element in?

Then perhaps a separate favicon PR if you do create a custom icon?

**Comment by meiamsome** [comment]
Closing in preference to #49

**Pull request closed without merging**